js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/CollapsibleSection.tsx

```diff
@@ -1,14 +1,15 @@
-import {Box, Colors, Icon, Subheading} from '@dagster-io/ui';
+import {Box, Colors, Icon, Subheading, Tooltip} from '@dagster-io/ui';
 import * as React from 'react';
 import styled from 'styled-components/macro';
 
 interface Props {
   header: React.ReactNode;
+  details: string;
```

Comment:
`details: React.ReactNode`

Reply:
Hmm I'm getting `Type 'ReactNode' is not assignable to type 'string | Element'.`

Reply:
switched to JSX.Element